### PR TITLE
Add some Portuguese translation

### PR DESCRIPTION
**Pull request by haileyok**
Can't vouch for top quality here but better than what we had. Hopefully can get cleaned up soon.

### CODE REVIEW

File: src/locale/locales/pt-BR/messages.po [hunk 1]
```diff
@@ -1626,7 +1626,7 @@ msgstr "Copiar texto do post"
 
 #: src/components/StarterPack/QrCodeDialog.tsx:171
 msgid "Copy QR code"
-msgstr ""
+msgstr "Copiar código QR "
```
Comment (iagocavalcante) [inline src/locale/locales/pt-BR/messages.po:1629]: For us its okay read "Copiar QR code"

**Review by iagocavalcante — COMMENTED** [review]
Just a small suggestion

**Review by pfrazee — APPROVED** [review]

File: src/locale/locales/pt-BR/messages.po (hunk 1 above)
Comment (haileyok) [reply src/locale/locales/pt-BR/messages.po:1629]: Thank you!! Feel free to leave more notes or anything too <3

Comment (haileyok) [reply src/locale/locales/pt-BR/messages.po:1629]:
```suggestion
msgstr "Copiar código QR "
```

**Pull request merged**